code/divide_conquer/fast_fourier_transform/FFT.java

```diff
@@ -0,0 +1,144 @@
+public class FFT {
```

Comment:
Please add open genus comment here !
You can refer to the community guidelines for the same ! 👍 
:)